### ISSUE TITLE
http://.... not found for my S9

### ISSUE THREAD

**Issue by carlcbilodeau** [anselal/antminer-monitor#57]
I have installed the antminer-monitor succesfully on my Windows 10. I have launch the app in my chrome browser and have added my S9 ip addresses. The problem is that the software adds http:// in front of the ip addresses and this makes the S9 not availlable. 

The software gives this STATUS message for each of my S9: Error: Check connection or IP Address 

What am I doing wrong? Should I edit my hosts file?

N.B.: In my web browser I can access let say a S9 like this:
10.0.0.51

but if I try

http://10.0.0.51 

it gives a dns error. So I suppose antminer-monitor should not use http:// in the Windows version.

**Comment by anselal** [comment]
Thank you for using my software. The app doesn't retrieve stats via http but via socket connection on port 4028. If you are behind a firewall/router you should open/forward port 4028.
HTTP is the default protocol for the web. What you describe makes no sense.

**Comment by carlcbilodeau** [comment]
I have found the solution. I edited the hosts in windows. Open notepad in "Administrator" mode. In notepad open the file c:\windows\system32\drivers\etc\hosts
At the end of this file, add one line for each Antminer machine like this:
10.0.0.61    http:\\10.0.0.61
10.0.0.62    http:\\10.0.0.62
.....

French:
Attention, je dois éditer avec Notepad le fichier hosts afin d’ajouter toutes mes S9. J’ouvre notepad en mode « administrateur ». Puis j’ouvre le fichier c:\windows\system32\drivers\etc\hosts
J’ajoute au bas les lignes pour chaque S9 comme suit :
10.0.0.61	http:\\10.0.0.61
